server/views/macros/adjudicationSummary.njk

```diff
@@ -19,7 +19,7 @@
     {% endif %}
   </div>
   <div class="adjudicationsSummary">
-    {{ summaryListWithChangeLinks(incidentData.incidentDetails, 'Not entered', { "data-qa": "details-summary-table" }) }}
+    {{ summaryListWithChangeLinks(incidentData.incidentDetails, 'Not entered', { "data-qa": "details-summary-table" }, '', "Report on someone's behalf") }}
```

Comment:
Really minor one - here, and in the tests you have `"Report on someone's behalf"` (double quotes) whereas for other string fields we use single quotes?
```suggestion
    {{ summaryListWithChangeLinks(incidentData.incidentDetails, 'Not entered', { "data-qa": "details-summary-table" }, '', 'Report on someone's behalf') }}
```

Reply:
ah it's because of the word used - "someone's" has an apostrophe so it breaks the string if i use single quotes

Reply:
Ah gotcha! will approve